test/opacity_comparison_funcs.jl

```diff
@@ -216,27 +216,17 @@ function Heminus_ff_coefficient(λ, T, Pₑ)
     # comparisons against plots from Gray (2005), we should employ the abundances from
     # their book. They record that there is 8.51e-2 He particles per H particle in table 16.3
     nHe = nH*8.51e-2
-    # we need to figure out the abundance of neutral Helium. We will use the ionization potentials
-    # tabulated in table D.1
-    χs = [24.587, NaN, NaN]
-    # technically, this should be χs = [24.587, 54.418] to account for both single and double
-    # ionization, but equation 8.16 in Gray (2005) only accounts for single ionization.
-
+    # Gray accounts for only neutral and singly ionized helium in his calculation.
+    # We use the ionization energy tabulated in table D.1 (χ(He I) = 24.587). 
     # Table D.2 of Gray (2005) records the partion function of He II to be
     # as 10^0.301, the closest value at the table's precision to 2.0
-    Us = [T -> 1.0,      # Table D.2 of Gray (2005) records the partion function of He II to be
-          T -> 10^0.301, # as 10^0.301, the closest value at the table's precision to 2.0
-          T -> NaN]
+    # Because this neglects He II, we don't use Korg.saha_ion_weights.
+    wII = (2.0/ne * (10^-0.301) * Korg.translational_U(Korg.electron_mass_cgs, T) 
+           * exp(-24.587/(Korg.kboltz_eV*T)))
+    nHe_I = nHe * 1/(1+wII)
```

Comment:
```suggestion
    UI, UII = (1.0, 10^-0.301)
    # Because this neglects He II, we don't use Korg.saha_ion_weights.
    wII = (2.0/ne * (UII/UI) * Korg.translational_U(Korg.electron_mass_cgs, T) 
           * exp(-24.587/(Korg.kboltz_eV*T)))
    nHe_I = nHe * 1/(1+wII)
```

I know that this is an extra line, but I think it improves readability to include `UI = 1.0`. (The alternative is including a comment reminding the reader that UI = 1.0, because things looked wrong)